ssl/d1_lib.c

```diff
@@ -865,15 +848,12 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client)
     if (BIO_dgram_get_peer(rbio, client) <= 0)
         BIO_ADDR_clear(client);
 
+    if (!dtls_buffer_listen_record(s, reclen, seq, align))
```

Comment:
I understand that this is an internal API and we can change it later if we need to, but it might be worth thinking about pulling up the processed_rcds reference to this call, to be clear that we're adding to the "already-processed" queue.

Reply:
It seems a little overkill to me to add it as a parameter. But I did add a comment before this line stating which queue is used.